salt/daemons/salting.py

```diff
@@ -3,7 +3,7 @@
 salting.py module of salt specific interfaces to raet
 
 '''
-from __future__ import absolute_import, print_function, unicode_literals
+from __future__ import absolute_import, print_function
```

Comment:
Why have you removed `unicode_literals`?

Reply:
It's because ioflo doesn't accept unicode. I've made it in the only RAET related modules. This helped to run it.

Reply:
I'll try to change this. Thank you for comments.